pilot/pkg/networking/core/v1alpha3/listener_builder.go

```diff
@@ -130,7 +130,7 @@ func reduceInboundListenerToFilterChains(listeners []*xdsapi.Listener) ([]*liste
 	return chains, needTLS
 }
 
-func (builder *ListenerBuilder) aggregateVirtualInboundListener() *ListenerBuilder {
+func (builder *ListenerBuilder) aggregateVirtualInboundListener(needTLSForPassThroughFilterChain bool) *ListenerBuilder {
```

Comment:
It might be better to follow the current naming convention in listener.go and listener_builder.go. The `needTLS` is used every where and I believe the `TLS` means the `tls_inspector`.